website/gatsby-browser.js

```diff
@@ -1,7 +1,6 @@
-import SmoothScroll from 'smooth-scroll';
-
 // Make scroll behavior of internal links smooth
 exports.onClientEntry = () => {
+  const SmoothScroll = require('smooth-scroll');
```

Comment:
https://next.gatsbyjs.org/docs/migrating-from-v1-to-v2/#convert-to-either-pure-commonjs-or-pure-es6

Reply:
OK. I still don't see why it should be moved from top-level, though, causing it to be imported each time the function is called instead of just once for the file?

Reply:
I think there was a build time issue because `window` was not defined. Not sure why it worked with gatsby v1. I'd have to test this again. The client entry should only happen once.

Reply:
A fix for the `window is not defined` bug is buried in the external media library PR: https://github.com/netlify/netlify-cms/pull/1602/files#diff-2d0cd5d10b9604941c38c6aac608178a